services/github/schemas/v1/taskcluster-render-yaml-input.yml

```diff
@@ -0,0 +1,76 @@
+$schema: "/schemas/common/metaschema.json#"
+title: "Render .taskcluster.yml input"
+description: |
+  Render .taskcluster.yml for given event.
+type:         object
+properties:
+  body:
+    title: Taskcluster.yml contents
+    description: The contents of the .taskcluster.yml
+    type: string
+  organization:
+    type: string
+    description: The organization that owns the repository.
+    pattern: "^[a-zA-Z0-9_-]+$"
+    default: TaskclusterRobot
+  repository:
+    type: string
+    description: The repository name.
+    pattern: "^[a-zA-Z0-9_-]+$"
+    default: hooks-testing
+  branch:
+    type: string
+    description: The branch name.
+    default: main
+  fakeEventType:
+    type: string
+    description: Name of the emulated event type
+    enum:
+      - github-push
+      - github-tag-push
+      - github-pull-request
+      - github-pull-request-untrusted
+      - github-release
+  fakePullRequestAction:
+    type: string
+    description: Name of the pull request action
+    default: opened
+    enum:
+      - opened
+      - synchronize
+      - reopened
+      - assigned
+      - auto_merge_disabled
+      - auto_merge_enabled
+      - closed
+      - converted_to_draft
+      - dequeued
+      - edited
+      - enqueued
+      - labeled
+      - ready_for_review
+      - review_requested
+      - review_request_removed
+      - unassigned
+      - unlabeled
+  fakeReleaseAction:
+    type: string
+    description: Name of the release action
+    default: published
+    enum:
+      - published
+      - unpublished
+      - created
+      - edited
+      - deleted
+      - prereleased
+      - released
+  fakeEventData:
+    type: object
+    description: |
+      Additional data to be added to the event object.
```

Comment:
could you provide more context here? when would you want to do this, what type of information can be provided? in a real situation if there was additional data here, where would it have come from? are there e.g. github docs that describe what additional data is allowed?

Reply:
👍 will add. 
If someone wants to send his test payload, instead of doing mocked one, this could be a way to send full event payload, that will be later passed to jsone context and rendered into tasks

Reply:
restructured payload, and added descriptions, it should be more clear now